docs/maintainer-guide/issues.md

```diff
@@ -0,0 +1,82 @@
+
+## Things to Keep in Mind
+
+1. **Be nice.** Even if the people are being rude or aggressive on an issue, as a project team member you must be the mature one in the conversation. Do your best to work with everyone no matter their style. Remember, poor wording choice can also be a sign of someone who doesn't know English very well, so be sure to consider that when trying to determine the tone of someone's message. Being rude, even when someone is being rude to you, reflects poorly on the team and the project as a whole.
+1. **Be inquisitive.** Ask questions on the issue whenever something isn't clear. Don't assume you understand what's being reported if there are details missing. Whenever you are unsure, it's best to ask for more information.
+1. **Not all requests are equal.** It's unlikely we'll be able to accommodate every request, so don't be afraid to say that something doesn't fit into the scope of the project or isn't practical. It's better to give such feedback if that's the case.
+1. **Close when appropriate.** Don't be afraid to close issues that you don't think will be done, or when it's become clear from the conversation that there's no further work to do. Issues can always be reopened if they are closed incorrectly, so feel free to close issues when appropriate. Just be sure to leave a comment explaining why the issue is being closed (if not closed by a commit).
+
+## Types of Issues
+
+There are four primary issue categories:
+
+1. **Bug** - something isn't working the way it's expected to work.
+1. **Enhancement** - a change to something that already exists. For instance, adding a new option to an existing rule.
+1. **Feature** - adding something that doesn't already exist. For example, adding a new rule, new formatter, or new command line flag.
+1. **Question** - an inquiry about how something works that won't result in a code change. We'd prefer if people use the mailing list or chatroom for questions, but sometimes they'll open an issue.
+
+The first goal when evaluating an issue is to determine which category the issue falls into.
+
+## When an Issue is Opened
+
+When a issue is opened, the bot will automatically apply the "triage" label. Issues labeled with "triage" are the ones that need to be looked at by team members to determine what to do next.
+
+The steps for triaging an issue are:
+
+1. Is it clear what is being requested?
+  * No: add the "needs info" label to the issue. The bot will add a comment asking for more information. You don't need to comment any further until the person who opened the issue responds with the information requested from the bot.
+  * Yes:
+    * Label questions with the "question" label
+    * Label bug reports with the "bug" label (also use the "accepted" label if you can reproduce and verify the bug)
```

Comment:
I don't know. I usually don't, because "proposal" seems like the wrong word when it comes to bugs. I added "proposal" for features and enhancements, but we can use it on bugs if you think it makes sense.